app/src/main/java/io/github/droidkaigi/confsched2017/view/transition/FadeOutTransition.java

```diff
@@ -0,0 +1,74 @@
+package io.github.droidkaigi.confsched2017.view.transition;
+
+import android.animation.Animator;
+import android.animation.ObjectAnimator;
+import android.animation.ValueAnimator;
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.graphics.Rect;
+import android.os.Build;
+import android.transition.Transition;
+import android.transition.TransitionValues;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * SharedElementTransition cannot Visibility transition like fade
+ * This transition can animate visibility on shared element transition.
+ */
+@TargetApi(Build.VERSION_CODES.KITKAT)
+public class FadeOutTransition extends Transition {
+    private static final String PROP_BOUNDS = "mdap:fabTransform:bounds";
+
+    private static final String[] transitionProperties = {
+            PROP_BOUNDS
+    };
+
+    public FadeOutTransition() {
+        super();
+    }
+
+    public FadeOutTransition(Context context, AttributeSet attrs) {
+        super(context, attrs);
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: Call requires API level 21 (current min is 19): new android.transition.Transition [NewApi]